plugins/data-management/src/AddTrackWidget/components/ConfirmTrack.tsx

```diff
@@ -98,6 +126,102 @@ function getTrackTypes(pluginManager: PluginManager) {
   return pluginManager.getElementTypesInGroup('track') as { name: string }[]
 }
 
+const TextIndexingConfig = observer(({ model }: { model: AddTrackModel }) => {
+  const classes = useStyles()
+  const [value1, setValue1] = useState('')
+  const [value2, setValue2] = useState('')
+  const [attributes, setAttributes] = useState(['Name', 'ID', 'type'])
```

Comment:
type is not something we need to index by default

Reply:
could use "Note" by default perhaps

Reply:
Since we do not have 'Note' as a default value in the indexing attributes for the track text searching conf, I would rather just keep 'Name' and 'ID'. Alternatively, we can change all the default values to include Note when indexing. https://github.com/GMOD/jbrowse-components/blob/main/packages/core/pluggableElementTypes/models/baseTrackConfig.ts#L41

Reply:
that's probably fine

Reply:
e.g. just keeping it as is